docs/_static/css/github_issue_links.css

```diff
@@ -0,0 +1,6 @@
+.github-issue-link-container {
+    padding-right: 0.5rem;
+}
+.github-issue-link {
+    font-size: 0.8125rem;
```

Comment:
I wonder where those numbers come from :D

Reply:
I saw it used elsewhere in the theme, but now I looked again and found `var(--font-size-small)` in the theme which is about the same but uses a variable which is cooler so I switched to that